LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC.
+Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC.
```

Comment:
```suggestion
Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC.
```

Perhaps it would make sense to run the update-copyright-year script again?